apps/dashboard/app/controllers/projects_controller.rb

```diff
@@ -125,11 +125,8 @@ def templates
       [label, project.directory, data]
     end
 
-    if templates.size.positive?
-      templates.prepend(['', '', { 'data-description': '', 'data-icon': '' }])
-    else
-      []
-    end
+    return templates if templates.size.positive?
+    return []
```

Comment:
I was going to say can we turn this into a ternary, but if it's empty, we may as well just return it right?  I mean it seems like we can just return `templates` and let the next layer deal with it being empty or not.